Makefile.doc

```diff
@@ -383,12 +383,18 @@ install-doc-index-urls:
 # Documentation of the source code (using ocamldoc)
 ###########################################################################
 
+# To skip building docs for plugins and build docs for core only, set this
+# variable to 1 (or any non-empty value):
+NOPLUGINDOCS ?=
```

Comment:
Why provide the option instead of always building plugins doc?

Reply:
No good reason, I've removed it.